datafusion/physical-expr/src/expressions/binary.rs

```diff
@@ -704,12 +721,12 @@ impl PartialEq<dyn Any> for BinaryExpr {
 // (on the right). The new boundaries will indicate whether it is always true, always
 // false, or unknown (with a probablistic selectivity value attached).
```

Comment:
I wonder if these docstrings could/should be updated based on the changed signature